src/sentry/integrations/vsts/issues.py

```diff
@@ -86,3 +88,45 @@ def get_issue(self, issue_id, **kwargs):
             'title': work_item['fields']['System.Title'],
             'description': work_item['fields'].get('System.Description')
         }
+
+    def sync_assignee_outbound(self, external_issue, user, assign=True, **kwargs):
+        client = self.get_client()
+        assignee = None
+
+        # TODO(LB): What's the scope here? is this correct?
+        # Get a list of all users in a given scope. How do we define scope?
+        # https://docs.microsoft.com/en-us/rest/api/vsts/graph/users/list?view=vsts-rest-4.1
```

Comment:
This appears correct, but there was no formal definition of scope. But it appears to be all users with access to the account, to the best of my knowledge.